hs_geographic_feature_resource/templates/pages/geographicfeatureresource.html

```diff
@@ -0,0 +1,235 @@
+{# This template is used as the landing page for NetCDF Resource #}
+
+{% extends "pages/genericresource.html" %}
+{% load geoanalytics_tags pages_tags mezzanine_tags crispy_forms_tags %}
+
+{% block extended_metadata %}
+        <div class="row">
+            <div class="col-md-12">
+                <div>
+                {# Add the view mode html code below for resource extended meta tab #}
+
+                {% if extended_metadata_exists %}
+
+                     {# Add the geometry information meta #}
+                    <h4 ><strong>Geometry Information:</strong></h4>
+                    {% if geometry_information %}
+                        <hr style="margin-top: 0px;margin-bottom: 2px">
+
+                        {% if geometry_information.geometryType %}
+                            <div class="row">
+                                <div class="col-sm-4"><strong>Geometry Type:</strong></div>
+                                <div class="col-sm-8">{{ geometry_information.geometryType}}</div>
```

Comment:
consistent spacing within `{{` variable tags `}}` is really helpful to readability.